src/pages/my_page/user_profile/UserRoleAndHelp.tsx

```diff
@@ -0,0 +1,118 @@
+
+export const UserRoleAndHelp = () => {
+  const { t } = useTranslation();
+  const user = useSelector((store: RootState) => store.user);
+  const customer = useSelector((store: RootState) => store.customer);
+  const customerServiceCenterUri = customer?.serviceCenterUri;
+
+  const username = user?.nvaUsername ?? '';
+
+  const nvaUserQuery = useQuery({
+    enabled: !!username,
+    queryKey: [username],
+    queryFn: () => fetchUser(username),
+    meta: { errorMessage: t('feedback.error.get_person') },
+  });
+
+  const nvaUser = nvaUserQuery.data;
+
+  return (
+    <BackgroundDiv sx={{ display: 'flex', flexDirection: 'column', gap: '1rem' }}>
+      {nvaUser?.viewingScope && (
+        <Box sx={{ gap: '0.75rem' }}>
+          <Typography gutterBottom fontWeight="bold">
+            {t('my_page.my_profile.my_area_of_responsibility')}
+          </Typography>
+          <Box
+            sx={{
+              display: 'flex',
+              gap: '0.5rem',
+              flexDirection: { xs: 'row', md: 'row' },
+              flexWrap: 'wrap',
+              mb: '0.5rem',
+            }}>
+            {nvaUser.viewingScope?.includedUnits.map((orgId) => (
+              <ViewingScopeChip key={orgId} organizationId={orgId} />
+            ))}
+          </Box>
+          <Divider sx={{ bgcolor: 'primary.main' }} />
+        </Box>
+      )}
+      {user && (
+        <>
+          <Box sx={{ maxWidth: '40rem' }}>
+            <UserRoles user={user} hasActiveEmployment={false} />
+          </Box>
+          <Divider sx={{ bgcolor: 'primary.main' }} />
+        </>
+      )}
+      <Typography variant="h3">{t('common.help')}</Typography>
+      <Box sx={{ display: 'flex', gap: '2rem' }}>
+        {customerServiceCenterUri && (
+          <>
+            <Box sx={{ display: 'flex', flexDirection: 'column' }}>
+              <Typography fontWeight="bold">
+                {t('my_page.my_profile.user_role_and_help.institution_help_page')}
+              </Typography>
+              <MuiLink
+                sx={{
+                  fontWeight: 'bold',
+                  color: 'primary.main',
+                  display: 'flex',
+                  gap: '0.5rem',
+                  py: '0.5rem',
+                  justifyContent: 'space-between',
+                }}
+                data-testid={dataTestId.myPage.userRolesAndHelp.institutionHelpPage}
+                target="_blank"
+                rel="noopener noreferrer"
+                href={customerServiceCenterUri}>
+                {customer?.displayName} {t('common.help').toLowerCase()}
+                <LaunchIcon fontSize="small" />
+              </MuiLink>
+              <Typography fontStyle="italic">
+                {t('my_page.my_profile.user_role_and_help.customer_help_text')}
+              </Typography>
+            </Box>
+            <Divider orientation="vertical" />
+          </>
+        )}
+        <Box sx={{ display: 'flex', flexDirection: 'column', maxWidth: '15rem' }}>
```

Comment:
virker overflødig med flex her? 🤔 
```suggestion
        <Box sx={{ maxWidth: '15rem' }}>
```

Reply:
Tok i bruk `StyledMuiLink`, så den er ikke overflødig lengre da.